flink-clients/src/test/java/org/apache/flink/client/CliFrontendSavepointTest.java

```diff
@@ -77,23 +74,25 @@
 	public void testTriggerSavepointSuccess() throws Exception {
 		replaceStdOutAndStdErr();
 
-		try {
-			JobID jobId = new JobID();
+		JobID jobId = new JobID();
 
-			String savepointPath = "expectedSavepointPath";
+		String savepointPath = "expectedSavepointPath";
 
-			MockedCliFrontend frontend = new SavepointTestCliFrontend(savepointPath);
+		final ClusterClient clusterClient = createClusterClient(savepointPath);
+
+		try {
+			MockedCliFrontend frontend = new MockedCliFrontend(clusterClient);
 
 			String[] parameters = { jobId.toString() };
-			int returnCode = frontend.savepoint(parameters);
+			frontend.savepoint(parameters);
 
-			assertEquals(0, returnCode);
```

Comment:
Because I moved the tests from a later commit where I change the signature of the `savepoint` command to return `void` and throw an `Exception` in case of a failure.  Will add the assertions again.